cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -538,29 +538,62 @@ function uploadAnnotationRequest() {
                     }
                     catch(error) {
                         overlay.remove();
-                        showMessage("Parsing errors was occured. " + error);
+                        showMessage("Parsing errors was occurred. " + error);
                         return;
                     }
 
-                    const exportData = createExportContainer();
-                    exportData.create = parsed;
-
                     let asyncSave = function() {
                         $.ajax({
-                            url: '/save/annotation/task/' + window.cvat.dashboard.taskID,
+                            url: '/delete/annotation/task/' + window.cvat.dashboard.taskID,
                             type: 'POST',
-                            data: JSON.stringify(exportData),
-                            contentType: 'application/json',
                             success: function() {
-                                let message = 'Annotation successfully uploaded';
-                                showMessage(message);
+                                asyncSaveChunk(0);
                             },
                             error: function(response) {
-                                let message = 'Annotation uploading errors was occured. ' + response.responseText;
+                                let message = 'Annotation uploading errors was occurred: ' +
```

Comment:
There is no data uploading. Clean up only.

Reply:
OK